docs/CODE_GUIDELINES.md

```diff
@@ -496,13 +552,22 @@ public:
 }
 ```
 
-### 8.6. Methods
+### 8.6. Structs
+Use PascalCase, the prefix with an uppercase S is not mandatory.
```

Comment:
```suggestion
Use PascalCase. `S` may be added as a prefix.
```
I'm not quite sure, do we actually want the `S`? Isn't it just there for legacy, in which case we should not have it in the guidelines at all?

Reply:
@notspiff has suggested add S prefix,
in our repository code there is a mixture of differents naming uses for Struct type, so it is not clear which method we use officially

I am also in favour of removing the prefix for struct if you see fit
more opinions would be appreciated

Reply:
IMO we should have same rules for classes and structs. If we enforce the usage of a `C` prefix in classes, `S` should also be enforced for structs.

If the decision is not to have a prefix for structs, why should we have them for classes? We've pretty much dropped all other types from variables

Reply:
CC @garbear @lrusak @thexai @notspiff

Reply:
> If the decision is not to have a prefix for structs, why should we have them for classes?

I don't see a logical reason why using `C` for classes should force you to use `S` for structs. For me, it's just a matter of policy and consistency. IMO, not having the `S` is totally fine, but I don't care a lot about this specific point.

Reply:
since currently on our source code the "structs" do not follow any guidelines and there are a lot mixed uses
i can omit the "S" as optional, and keep only PascalCase
at least as a basis to follow

Reply:
sounds fine to me - @notspiff?

Reply:
for now removed and kept only pascalcase